pkgs/development/misc/brev-cli/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "brev-cli";
+  version = "0.6.95";
+
+  src = fetchFromGitHub {
+    owner = "brevdev";
+    repo = pname;
```

Comment:
```suggestion
    repo = "brev-cli";
```

Reply:
I am a bit curious about the rationale behind this. Isn't this a pretty common pattern in nixpkgs?

Reply:
Yeah, it is pretty common but it is technically not really correct and can cause issues when pname is updated which is arguable only common in pythonPackages.